client/components/jetpack/daily-backup-status/status-card/backup-successful.jsx

```diff
@@ -56,6 +56,12 @@ const BackupSuccessful = ( { backup, deltas, selectedDate } ) => {
 		( 'rewind__backup_complete_full' !== backup.activityName ||
 			'rewind__backup_only_complete_full' !== backup.activityName );
 
+	// If the activity isn't rewindable but has streams that are,
+	// use one of the streams' rewind ID as the download or restore point
+	const actionableRewindId = backup.activityIsRewindable
+		? backup.rewindId
+		: backup.streams?.find?.( ( a ) => a.activityIsRewindable )?.rewindId;
```

Comment:
Same as above, though if we're reusing this code would it make sense to be a hook?

Reply:
I'm not opposed to making this into a hook or otherwise DRYing it up. Out of curiosity, do we have a standard convention for where to put hooks, like a `/hooks` directory? My hunch is no, but I thought I'd ask to be sure.

Reply:
Not that I'm aware of! I believe we put a bunch in `lib/jetpack` however?